docs/index.md

```diff
@@ -0,0 +1,198 @@
+---
+file_format: mystnb
+kernelspec:
+  name: python3
+---
+# Qermit
+
+Qermit is a python module for running error-mitigation protocols.
+It contains an assortment of ready to use error-mitigation schemes.
+Qermit supports straightforward composition of these existing
+error-mitigation methods, and the development of new error-mitigation
+schemes.
+
+Qermit is an extension to the pytket quantum software development kit. 
+Qermit uses the pytket {py:class}`~pytket.backends.Backend` class,
+and so supports an array of backends.
+
+## Getting Started
+
+```{code-block} console
+pip install qermit
+```
+
+Please visit the
+[github repository](https://github.com/CQCL/Qermit/issues) if you would
+like to install from source. We also welcome contributions to Qermit there!
+
+If you notice any bugs while using Qermit we would much appreciate you
+raising them as issues at that repository.
+
+If you would like an further
+support with Qermit please contact <tket-support@quantinuum.com>.
+
+## Key Notions
+
+Error-mitigation methods in Qermit are either one of two types:
+
+- {py:class}`~qermit.taskgraph.mitres.MitRes` methods which modify a distribution of counts.
+- {py:class}`~qermit.taskgraph.mitex.MitEx` methods which modify the expectation value of some observable.
+
+While Qermit includes many advanced error-mitigation schemes,
+in their basic capacity, {py:class}`~qermit.taskgraph.mitres.MitRes` and
+{py:class}`~qermit.taskgraph.mitex.MitEx` objects will run
+experiments without error-mitigation, as in the following example.
+
+```{code-cell} ipython3
+from qermit import MitRes, CircuitShots
+from pytket import Circuit
+from pytket.extensions.qiskit import AerBackend
+
+# Define the experiment to be run.
+# In this case, a Bell pair is prepared and measured 1000 times.
+circ = Circuit(2,2).H(0).CX(0,1).measure_all()
+circ_shots = CircuitShots(Circuit=circ, Shots=1000)
+
+# Define the way the experiment will be run.
+# In this case the default MitRes, without error-mitigation,
+# will be performed.
+mitres = MitRes(backend = AerBackend())
+
+# Finally, run the experiment.
+results = mitres.run([circ_shots])
+results[0].get_counts()
+```
+
+Here we have introduced the {py:obj}`~qermit.taskgraph.mittask.CircuitShots`.
+This can be thought of a defining experiment to run to generated a
+distribution of shots. In this case the Bell state preparation circuit will
+be run 1000 times to generate the desired distribution. The
+{py:class}`~qermit.taskgraph.mitres.MitRes` definition can then be thought
+of as specifying how the experiment will run, in this case without
+error-mitigation,
+and by using {py:class}`~pytket.extensions.qiskit.AerBackend`.
```

Comment:
Sorry to revive a merged PR @CalMacCQ, but I note that intersphinx does not work with `AerBackend`. Is there a way to fix this that you know of?